dark-mode-screenshot.js

```diff
@@ -7,49 +7,28 @@ const commandLineArgs = require('command-line-args');
   // Parse CLI arguments and set defaults.
   const optionDefinitions = [
     {name: 'url', alias: 'u', type: String, defaultValue:
-        'https://googlechromelabs.github.io/dark-mode-toggle/demo/index.html'},
+        'https://googlechromelabs.github.io/dark-mode-toggle/demo/'},
     {name: 'output', alias: 'o', type: String, defaultValue: 'screenshot'},
     {name: 'fullPage', alias: 'f', type: Boolean, defaultValue: true},
   ];
   const arguments = commandLineArgs(optionDefinitions);
   const {url, output, fullPage} = arguments;
-  // TODO: Once https://github.com/GoogleChrome/puppeteer/issues/4752 is
-  // resolved, make it run headless.
-  let options = {headless: false};
 
-  // Check that Dark Mode is on, else, this script can't work.
-  let browser = await puppeteer.launch(options);
-  let page = await browser.newPage();
-  const darkModeOn = await page.evaluate(() => {
-    return window.matchMedia('(prefers-color-scheme: dark)').matches;
+  const browser = await puppeteer.launch({
+    // TODO: Once https://github.com/GoogleChrome/puppeteer/issues/4752 is
+    // resolved, make it run headless.
+    headless: false,
   });
-  if (!darkModeOn) {
-    console.error('Dark Mode needs to be on for this script to work.',
-        'Please turn it on first.\n👉', 'https://goo.gle/activate-dark-mode');
-    await browser.close();
-    return;
-  }
-  await browser.close();
-  console.log('✅ Dark Mode is on, ready to go…');
+  const page = await browser.newPage();
+  await page.goto(url);
 
   // Create Light Mode screenshot
-  options.args = [
-    '--disable-blink-features=MediaQueryPrefersColorScheme'
-  ];
-  browser = await puppeteer.launch(options);
-  page = await browser.newPage();
-  await page.goto(url);
+  await page.emulateMediaFeatures([{ name: 'prefers-color-scheme', value: 'light' }]);
   await page.screenshot({path: `${output}-light.png`, fullPage: fullPage});
-  await browser.close();
   console.log('🌞 Created Light Mode screenshot.');
 
   // Create Dark Mode screenshot
-  options.args = [
-    '--force-dark-mode'
-  ];
-  browser = await puppeteer.launch(options);
-  page = await browser.newPage();
-  await page.goto(url);
+  await page.emulateMediaFeatures([{ name: 'prefers-color-scheme', value: 'dark' }]);
   await page.screenshot({path: `${output}-dark.png`, fullPage: fullPage});
```

Comment:
Note: because the demo uses a CSS transition, you'll need to wait for the transition to complete before taking the screenshot. Currently, you'd get a screenshot taken during the transition phase. Puppeteer is simply too fast now that it no longer needs a separate browser instance ;)